<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>633</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>633</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>9999</width>
    <height>9999</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>操作界面</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="minimumSize">
       <size>
        <width>1024</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>50</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1,1,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="2">
        <widget class="QLabel" name="img_label">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(:/Img/Admin.jpg);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="title_label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>学生信息管理系统</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="3">
           <widget class="QLabel" name="user_label">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>用户名：Admin</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>175</width>
        <height>500</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="widget_3" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>500</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <widget class="QWidget" name="widget_5" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <widget class="QPushButton" name="search_btn">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="contextMenuPolicy">
             <enum>Qt::DefaultContextMenu</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="src.qrc">
              <normaloff>:/Img/Search.png</normaloff>:/Img/Search.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLineEdit" name="search_lineEdit">
            <property name="placeholderText">
             <string>依据姓名匹配</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>530</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>400</height>
              </size>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>序号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>姓名</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>年龄</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>年级</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>班级</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>学号</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>电话</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>微信</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="num_label">
             <property name="styleSheet">
              <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;,rgb(255, 0, 0);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_add">
   <property name="text">
    <string>增添</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="text">
    <string>Del</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionModify">
   <property name="text">
    <string>Modify</string>
   </property>
  </action>
  <action name="actionby_name">
   <property name="text">
    <string>by_name</string>
   </property>
  </action>
  <action name="actionby_stu_id">
   <property name="text">
    <string>by_stu_id</string>
   </property>
  </action>
  <action name="action_del">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="action_search">
   <property name="text">
    <string>查询</string>
   </property>
  </action>
  <action name="action_modify">
   <property name="text">
    <string>修改</string>
   </property>
  </action>
  <action name="action_sort">
   <property name="text">
    <string>排序</string>
   </property>
  </action>
  <action name="action_destory">
   <property name="text">
    <string>清除</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
